src/group_theory/sylow.lean

```diff
@@ -53,93 +46,6 @@ by rw [← fintype.card_prod, fintype.card_congr
 
 namespace sylow
 
-/-- Given a vector `v` of length `n`, make a vector of length `n+1` whose product is `1`,
-by consing the the inverse of the product of `v`. -/
-def mk_vector_prod_eq_one (n : ℕ) (v : vector G n) : vector G (n+1) :=
```

Comment:
Did you drop all these declarations intentionally?

Reply:
See previous comment.